content/en/data_jobs/airflow.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Enable Data Jobs Monitoring for Apache Airflow
+is_beta: true
+private: true
+aliases:
+    - /data_jobs/airflow/
+    - data_jobs/airflow/
+further_reading:
+    - link: '/data_jobs'
+      tag: 'Documentation'
+      text: 'Data Jobs Monitoring'
+---
+
+{{< callout url="#" header="false" btn_hidden="true" >}}
+  Data Jobs Monitoring for Apache Airflow is in preview.
+{{< /callout >}}
+
+[Data Jobs Monitoring][1] gives visibility into the performance and reliability of workflows run by Apache Airflow DAGs.
+
+{{< tabs >}}
+{{% tab "Kubernetes" %}}
+## Requirements
+
+* [Apache Airflow 2.8.3][1] or later is required.
+* [apache-airflow-providers-openlineage 1.11.0][2] or later is required.
+
+## Setup
+
+Data Jobs Monitoring is supported for Apache Airflow deployment with [apache-airflow-providers-openlineage][2] installed.
+
+To get started, follow the instructions below.
+
+1. Install `openlineage` provider by adding the following into your `requirements.txt` file or wherever your Airflow depedencies are managed:
+   
+   ```text
+   apache-airflow-providers-openlineage>=1.11.0
+   ```
+
+2. Configure `openlineage` provider. The simplest option is to set the following environment variables and make them available to pods where you run Airflow schedulers and Airflow workers:
+   
+   ```shell
+   OPENLINEAGE_URL=<DD_DATA_OBSERVABILITY_INTAKE>
+   OPENLINEAGE_API_KEY=<DD_API_KEY>
+   ```
+
+   **Important:**
+   * install and configure `openlineage` provider for both Airflow schedulers and Airflow workers.
+   * replace `<DD_DATA_OBSERVABILITY_INTAKE>` fully with `https://data-obs-intake.`{{< region-param key="dd_site" code="true" >}}.
+   * replace `<DD_API_KEY>` fully with your valid [Datadog API key][4].
+   
+   **Optionally:**
+   * set `AIRFLOW__OPENLINEAGE__NAMESPACE` with a unique name for your Airflow instance to allow jobs from different Airflow instances logically separated.
```

Comment:
I don't think there is a consistent naming of this, I think you callout make sense here, it would be more rationale to call it deployment, that's what Astronomer call them just FYI.